README.md

```diff
@@ -2,30 +2,86 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Develop an **Entity Relationship Data Model** for a specific problem domain.
+- Use `dbdiagram.io` to create an **Entity Relationship Diagram (ERD)**.
+- Use Java Persistence API (JPA) to implement the entities and relationships.
 
-## Introduction
+## Setup
 
-A 1-2 sentence summary of what will be covered.
+1. [Fork and clone](https://github.com/learn-co-curriculum/java-mod-5-data-modeling-lab) the starter project.  
+   - The `pom.xml` file contains the Hibernate and PostgreSQL dependencies.
+   - The `org.example` directory contains `JpaCreate` and `JpaRead` classes with empty `main` methods.
+   - The `org.example.model` directory is empty.  Add your JPA entity classes in this directory.
+   - The `persistence.xml` file is configured to connect to `erd_lab_db`.
+2. Use the `pgAdmin` application to create a new database named `erd_lab_db`.
 
 ## Instructions
 
-Walk the student through any setup required to run the lesson (i.e.
-`npm install` and `npm start`).
+Pick one of the following problem domains:
 
-## Deliverables
+1. Hotel reservation database:  
+   A hotel has rooms available for customers to reserve. Data about each room includes
+   the room number, number and size of beds (i.e. full, queen, king), and whether it is a
+   pet-friendly room.
+   A customer makes a reservation to reserve one or more hotel rooms
+   for a specific arrival and departure date.
+   A customer must provide their name, email, phone number, and credit/debit card number
+   to make a reservation.  Customers may make many reservations, but each reservation is for a single customer.
+2. Vehicle repair appointment database:    
+   An auto shop takes appointments to repair vehicles. 
+   Data about each vehicle includes the make, model, year, and VIN.  An appointment
+   is scheduled for one vehicle at a particular date and time, and
+   a comment about the intended repair is included with the appointment.
+   One or more mechanics may work on the vehicle during the appointment.  Data about each
+   mechanic includes their name, date of hire, and social security number.
+3. Airline flight scheduling database:    
+   An airline flight has a scheduled departure and arrival data and time, and reserves
```

Comment:
Typo: "... departure and arrival data and time". I believe "data" should actually be "date"

Reply:
yes, thanks